gusto/linear_solvers.py

```diff
@@ -136,14 +139,18 @@ def _setup_solver(self):
         def V(u):
             return k*inner(u,k)
 
+        # specify degree for some terms as estimated degree is too large
+        dxp = dx(degree=(5,5))
```

Comment:
You mean make it an optional argument to the linear solver? I chose (5,5) based on the automatic quadrature degree that ufl was choosing, and also as it is what we used to use the slice models code. I'm just updating the docstrings in this file and I can't remember why degree=(5,5) and not just 5 - can you? or @wence- ?